app/models/client_opportunity_match.rb

```diff
@@ -514,7 +521,9 @@ def succeeded!
         opportunity_related_matches.each do |match|
           if match.active
             opportunity.notify_contacts_of_success(self)
-            MatchEvents::DecisionAction.create(match_id: match.id,
+            MatchEvents::DecisionAction.create(
```

Comment:
Same as with `canceled_events` if we add a `has_many: :decision_action_events` we can take advantage of the relationship to create the event.